### PR TITLE
Reduce style related warnings

### PR DESCRIPTION
**Pull request by zoq**
Reduce style related warnings in the build process; example: https://travis-ci.org/mlpack/mlpack/jobs/570648288

### CODE REVIEW

File: src/mlpack/core/data/scaler_methods/pca_whitening.hpp [hunk 1]
```diff
@@ -88,7 +88,7 @@ class PCAWhitening
     if (eigenValues.is_empty() || eigenVectors.is_empty())
     {
       throw std::runtime_error("Call Fit() before Transform(), please"
-        " refer documentation.");
+          " refer documentation.");
```
Comment (rcurtin) [inline src/mlpack/core/data/scaler_methods/pca_whitening.hpp]: I guess this should say `refer to the` if you want to fix that also. :)

File: src/mlpack/tests/augmented_rnns_tasks_test.cpp [hunk 2]
```diff
@@ -95,7 +95,8 @@ class HardCodedSortModel
   void Train(arma::field<arma::mat>& predictors,
              arma::field<arma::mat>& labels)
   {
-    assert(predictors.n_elem == labels.n_elem);
+    const bool check = predictors.n_elem == labels.n_elem;
+    assert(check == true);
```
Comment (rcurtin) [inline src/mlpack/tests/augmented_rnns_tasks_test.cpp]: Interesting, I guess this one is a little tricker---I wonder if just using `Log::Assert()` could fix the unused variable warning.

File: src/mlpack/tests/augmented_rnns_tasks_test.cpp [hunk 3]
```diff
@@ -95,7 +95,7 @@ class HardCodedSortModel
   void Train(arma::field<arma::mat>& predictors,
              arma::field<arma::mat>& labels)
   {
-    assert(predictors.n_elem == labels.n_elem);
+    Log::Assert(predictors.n_elem == labels.n_elem);
```
Comment (ShikharJ) [inline src/mlpack/tests/augmented_rnns_tasks_test.cpp]: I guess `Log` isn't declared here.

**Comment by rcurtin** [comment]
The build failure was just an AppVeyor timeout, so I saw no issue with merging. :)

**Pull request merged**